app/views/volunteers/edit.html.erb

```diff
@@ -1,5 +1,30 @@
-<div>
-    <h1>Update My Profile</h1>
+<style>
+
+    .body {
+        margin-top: 250px;
+        margin-left: 25px;
+        margin-right: 25px;
+    }
+    
+    .edit_title {
+        text-align: center;
+    }
+         
+</style>
+
+<div class='fixed-top'>
+    <%= render 'banner' %>
+    <%= render 'navbar' %>
 </div>
 
-<%= render 'form' %>
\ No newline at end of file
+<div class='body'>
+    
+    <div class='edit_title'>
+        <h2>UPDATE MY PROFILE</h2>
+    </div>
+    
+    <%= render 'form' %>
+</div>
+<div class='fixed-bottom'>
+    <%= render 'footer' %>
+</div>
```

Comment:
The footer is already wrapped inside of `<footer>` tags, so I don't think that the surrounding `<div>` tags are helpful here.